lib/cog/command/ambiguous_alias_checker.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Cog.Command.AmbiguousAliasChecker do
+  alias Cog.Repo
+  alias Cog.Queries.Alias
+  alias Cog.Models.Command
+  alias Piper.Command.Ast
+
+  def ambiguous_command?(%Ast.Invocation{command: "site:" <> _}),
+    do: false
+  def ambiguous_command?(%Ast.Invocation{command: "user:" <> _}),
```

Comment:
You could do this by adding a custom validation function to the flow in `Cog.Models.Bundle.changeset/2`, similar to what I did to enforce the fact that the `operable` bundle should always be enabled, for example.